browser_patches/firefox/juggler/NetworkObserver.js

```diff
@@ -156,7 +227,7 @@ class NetworkRequest {
 
     httpChannel.QueryInterface(Ci.nsITraceableChannel);
     this._originalListener = httpChannel.setNewListener(this);
-    if (redirectedFrom) {
+    if (redirectedFrom && !redirectedFrom._responseInterceptor) {
```

Comment:
This needs a comment - why redirects from the interception have different behavior.

Reply:
Done.